### PR TITLE
workflow: Remove blue hyperlinks in README

### PR DESCRIPTION
**Pull request by andrewwormald**
Before:
Screenshot 2024-08-09 at 14 22 37

After:
Screenshot 2024-08-09 at 14 22 28

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,15 @@
...
-        </a>
+        <img src="https://goreportcard.com/badge/github.com/luno/workflow"/> 
+        <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=coverage"/>
+        <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=alert_status"/>
+        <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=bugs"/>
+        <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=reliability_rating"/>
+        <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=security_rating"/>
+        <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=sqale_rating"/>
+        <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=vulnerabilities"/>
+        <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=duplicated_lines_density"/>
```
Comment (echarrod) [inline README.md:12]: I quite liked how it took you to https://sonarcloud.io/summary/new_code?id=luno_workflow when you clicked on the badges! Is it just the display of that you don't like?

Comment (andrewwormald) [reply README.md:12]: Okay let me try another way as I liked that too

Comment (andrewwormald) [reply README.md:12]: I think i can remove the test decorator...

Comment (andrewwormald) [reply README.md]: Fixed now with the links remaining 😁

**Review by echarrod — APPROVED** [review]
Perfect 👌

**Pull request merged**